### ISSUE TITLE
Epubcheck playOrder errors 

### ISSUE THREAD

**Issue by belldandu** [dteviot/WebToEpub#20]
This is the last remaining error that i have so far been unable to fix
```
[kami@Index ~]$ epubcheck Toaru_Maju...dexVolume1.epub 
Validating using EPUB version 2.0.1 rules.
ERROR(RSC-005): Toaru_Maju...dexVolume1.epub/OEBPS/toc.ncx(1,884): Error while parsing file 'different playOrder values for navPoint/navTarget/pageTarget that refer to same target'.
ERROR(RSC-005): Toaru_Maju...dexVolume1.epub/OEBPS/toc.ncx(1,1080): Error while parsing file 'different playOrder values for navPoint/navTarget/pageTarget that refer to same target'.
ERROR(RSC-005): Toaru_Maju...dexVolume1.epub/OEBPS/toc.ncx(1,2006): Error while parsing file 'different playOrder values for navPoint/navTarget/pageTarget that refer to same target'.
ERROR(RSC-005): Toaru_Maju...dexVolume1.epub/OEBPS/toc.ncx(1,2191): Error while parsing file 'different playOrder values for navPoint/navTarget/pageTarget that refer to same target'.
ERROR(RSC-005): Toaru_Maju...dexVolume1.epub/OEBPS/toc.ncx(1,2735): Error while parsing file 'different playOrder values for navPoint/navTarget/pageTarget that refer to same target'.
ERROR(RSC-005): Toaru_Maju...dexVolume1.epub/OEBPS/toc.ncx(1,2923): Error while parsing file 'different playOrder values for navPoint/navTarget/pageTarget that refer to same target'.

Check finished with errors

epubcheck completed
```

This is the toc.ncx
```
<?xml version='1.0' encoding='utf-8'?>
<ncx xmlns="http://www.daisy.org/z3986/2005/ncx/" version="2005-1" xml:lang="en">

    <head>
        <meta content="https://web.archive.org/web/20140803022146/http://www.baka-tsuki.org/project/index.php?title=Toaru_Majutsu_no_Index:Volume1" name="dtb:uid" />
        <meta content="2" name="dtb:depth" />
        <meta content="0" name="dtb:totalPageCount" />
        <meta content="0" name="dtb:maxPageNumber" />
    </head>
    <docTitle><text>Toaru Majutsu no Index:Volume1</text></docTitle>
    <navMap>
        <navPoint id="body0001" playOrder="1">
            <navLabel><text>Novel Illustrations</text></navLabel>
            <content src="Text/0000_Novel_Illustrations.xhtml" />
        </navPoint>
        <navPoint id="body0002" playOrder="2">
            <navLabel><text>Prologue: The Tale of the Illusion Killer Boy. The_Imagine-Breaker.</text></navLabel>
            <content src="Text/0001_Prologue_T...ne-Breaker.xhtml" />
        </navPoint>
        <navPoint id="body0003" playOrder="3">
            <navLabel><text>Chapter 1: The Magician Lands on the Tower. FAIR,_Occasionally_GIRL.</text></navLabel>
            <content src="Text/0002_Chapter_1_...nally_GIRL.xhtml" />
            <navPoint id="body0004" playOrder="4">
                <navLabel><text>Part 1</text></navLabel>
                <content src="Text/0002_Chapter_1_...nally_GIRL.xhtml" />
            </navPoint>
            <navPoint id="body0005" playOrder="5">
                <navLabel><text>Part 2</text></navLabel>
                <content src="Text/0003_Part_2.xhtml" />
            </navPoint>
            <navPoint id="body0006" playOrder="6">
                <navLabel><text>Part 3</text></navLabel>
                <content src="Text/0004_Part_3.xhtml" />
            </navPoint>
            <navPoint id="body0007" playOrder="7">
                <navLabel><text>Part 4</text></navLabel>
                <content src="Text/0005_Part_4.xhtml" />
            </navPoint>
            <navPoint id="body0008" playOrder="8">
                <navLabel><text>Part 5</text></navLabel>
                <content src="Text/0006_Part_5.xhtml" />
            </navPoint>
            <navPoint id="body0009" playOrder="9">
                <navLabel><text>Part 6</text></navLabel>
                <content src="Text/0007_Part_6.xhtml" />
            </navPoint>
            <navPoint id="body0010" playOrder="10">
                <navLabel><text>Part 7</text></navLabel>
                <content src="Text/0008_Part_7.xhtml" />
            </navPoint>
        </navPoint>
        <navPoint id="body0011" playOrder="11">
            <navLabel><text>Chapter 2: The Illusionist Bestows Demise. The_7th-Egde.</text></navLabel>
            <content src="Text/0009_Chapter_2_...e_7th-Egde.xhtml" />
            <navPoint id="body0012" playOrder="12">
                <navLabel><text>Part 1</text></navLabel>
                <content src="Text/0009_Chapter_2_...e_7th-Egde.xhtml" />
            </navPoint>
            <navPoint id="body0013" playOrder="13">
                <navLabel><text>Part 2</text></navLabel>
                <content src="Text/0010_Part_2.xhtml" />
            </navPoint>
            <navPoint id="body0014" playOrder="14">
                <navLabel><text>Part 3</text></navLabel>
                <content src="Text/0011_Part_3.xhtml" />
            </navPoint>
            <navPoint id="body0015" playOrder="15">
                <navLabel><text>Part 4</text></navLabel>
                <content src="Text/0012_Part_4.xhtml" />
            </navPoint>
        </navPoint>
        <navPoint id="body0016" playOrder="16">
            <navLabel><text>Chapter 3: The Grimoire Peacefully Smiles. "Forget_me_not."</text></navLabel>
            <content src="Text/0013_Chapter_3_...get_me_not.xhtml" />
            <navPoint id="body0017" playOrder="17">
                <navLabel><text>Part 1</text></navLabel>
                <content src="Text/0013_Chapter_3_...get_me_not.xhtml" />
            </navPoint>
            <navPoint id="body0018" playOrder="18">
                <navLabel><text>Part 2</text></navLabel>
                <content src="Text/0014_Part_2.xhtml" />
            </navPoint>
            <navPoint id="body0019" playOrder="19">
                <navLabel><text>Part 3</text></navLabel>
                <content src="Text/0015_Part_3.xhtml" />
            </navPoint>
            <navPoint id="body0020" playOrder="20">
                <navLabel><text>Part 4</text></navLabel>
                <content src="Text/0016_Part_4.xhtml" />
            </navPoint>
        </navPoint>
        <navPoint id="body0021" playOrder="21">
            <navLabel><text>Chapter 4: The Exorcist Chooses the End. (N)Ever_Say_Good_bye.</text></navLabel>
            <content src="Text/0017_Chapter_4_...y_Good_bye.xhtml" />
        </navPoint>
        <navPoint id="body0022" playOrder="22">
            <navLabel><text>Epilogue: The Conclusion of the Index of Prohibited Books Girl. Index-Librorum-Prohibitorum.</text></navLabel>
            <content src="Text/0018_Epilogue_T...ohibitorum.xhtml" />
        </navPoint>
        <navPoint id="body0023" playOrder="23">
            <navLabel><text>Afterword</text></navLabel>
            <content src="Text/0019_Afterword.xhtml" />
            <navPoint id="body0024" playOrder="24">
                <navLabel><text>Translator's Notes</text></navLabel>
                <content src="Text/0020_Translators_Notes.xhtml" />
            </navPoint>
            <navPoint id="body0025" playOrder="25">
                <navLabel><text>Alternate Translations</text></navLabel>
                <content src="Text/0021_Alternate_...anslations.xhtml" />
            </navPoint>
        </navPoint>
    </navMap>
</ncx>
```
This is probably the only thing so far i'm stumped by. If anyone wants to enlighten me as to whats wrong here as far as the playOrder is concerned i'm all ears (and eyes).

**Comment by dteviot** [comment]
I think part of your problem is the toc.ncx is stored as a single line.  Hence the line numbers in the output are all 1.  If you were to reformat the toc.ncx with line breaks and put it back in the epub.  The line numbers should tell you where the conflicts are.

**Comment by dteviot** [comment]
Actually, I think it's complaining about things like this.
```xml
            <navLabel><text>Chapter 1: The Magician Lands on the Tower. FAIR,_Occasionally_GIRL.</text></navLabel>
            <content src="Text/0002_Chapter_1_...nally_GIRL.xhtml" />
            <navPoint id="body0004" playOrder="4">
                <navLabel><text>Part 1</text></navLabel>
                <content src="Text/0002_Chapter_1_...nally_GIRL.xhtml" />
            </navPoint>
```
The two items have the same <content src=, because they point to the same page.

**Comment by belldandu** [comment]
Yeah it is. Gonna see if i can do something about this (maybe) 

I could possibly check if the src of the current navpoint  === the src of the last navpoint and if so then don't update the play order.

**Comment by belldandu** [comment]
Yeah there we go fixed in #19
